docs/components/table-view.md

```diff
@@ -743,6 +817,99 @@ This slot is only available when the `actions` header key is present in [`header
 </KTableView>
 ```
 
+### bulk-action-items
+
+Slot for passing bulk action dropdown items.
```

Comment:
question: what happens if the user enables bulkActions but doesn't provide any dropdown actions? Is there a way to check if the slot has content and if not, force bulk actions to be disabled? How should this be handled?